module/VuFind/src/VuFind/Search/Factory/AbstractSolrBackendFactory.php

```diff
@@ -339,11 +339,13 @@ protected function getHiddenFilters()
     protected function createConnector()
     {
         $config = $this->config->get($this->mainConfig);
+        $searchConfig = $this->config->get($this->searchConfig);
+        $defaultFields = $searchConfig->General->default_record_fields ?? '*,score';
```

Comment:
How about dropping score from all the defaults? As far as I can see it's not actually used anywhere.

Reply:
I think you're probably correct; I don't see any explicit use of it anywhere. As you noted elsewhere, it would eliminate score from display in staff view of non-MARC records, but I think that's the only user-facing change (unless somebody has custom record driver methods locally defined that rely on it, which seems unlikely but not impossible).

I suppose one option would be to remove "score" in one place but not the other, to demonstrate how the .ini file can override the core defaults -- for example, change it in the hard-coded PHP default, and then override it in the .ini files to maintain legacy behavior and offer a tuning option... or, alternatively, leave it alone in the PHP, but show a commented out option in the .ini files that omits score so that people can make the change as an optimization if they wish. I don't really have a strong opinion about the best approach, but this is definitely in the category of things that have been this way for many years, so changing them has some potential for unintended consequences.